arraycontext/impl/pytato/compile.py

```diff
@@ -230,17 +233,22 @@ def _as_dict_of_named_arrays(keys, ary):
         pt_dict_of_named_arrays = self.actx.transform_dag(
             pt.make_dict_of_named_arrays(dict_of_named_arrays))
 
-        pytato_program = pt.generate_loopy(pt_dict_of_named_arrays,
-                                           options=lp.Options(
-                                               return_dict=True,
-                                               no_numpy=True),
-                                           cl_device=self.actx.queue.device)
-        assert isinstance(pytato_program, BoundPyOpenCLProgram)
-
-        pytato_program = (pytato_program
-                          .with_transformed_program(self
-                                                    .actx
-                                                    .transform_loopy_program))
+        from pytools import ProcessLogger
+        with ProcessLogger(logger, "generate_loopy"):
+            pytato_program = pt.generate_loopy(pt_dict_of_named_arrays,
+                                               options=lp.Options(
+                                                   return_dict=True,
+                                                   no_numpy=True),
+                                               cl_device=self.actx.queue.device)
+            assert isinstance(pytato_program, BoundPyOpenCLProgram)
+
+        print(pytato_program.program)
```

Comment:
```suggestion
        logger.debug(pytato_program.program)
```
? In case this was intentional?